recipes/attrs-sqlalchemy/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "attrs_sqlalchemy" %}
```

Comment:
@vamega the directory name is `attrs-sqlalchemy`, but the recipe name is `attrs_sqlalchemy`. They differ by `-`/`_`. Can you please send a PR to use the same character in both cases?